includes/class-freemius.php

```diff
@@ -240,7 +240,7 @@ private function __construct( $slug ) {
 
 			$this->_plugin_main_file_path = $this->_find_caller_plugin_file();
 			$this->_plugin_dir_path       = plugin_dir_path( $this->_plugin_main_file_path );
-			$this->_plugin_basename       = plugin_basename( $this->_plugin_main_file_path );
+			$this->_plugin_basename       = $this->_get_plugin_basename( $this->_plugin_main_file_path );
```

Comment:
@fajardoleo When a function is actually private, there's no need to add the '_' (underscore) prefix. For example in:
`private function _get_plugin_basename`

The underscore should be added to functions that supposed to be private (not used outside the SDK), but we had to make them public due to different reasons. For example, when the function is called with a filter/action, there's no way to keep it private.

You might find some functions that have the `private` directive but also have an underscore, it's my mistake.